modules/rules_python/metadata.json

```diff
@@ -21,9 +21,8 @@
         "0.12.0",
         "0.14.0",
         "0.15.0",
-        "0.15.1"
+        "0.15.1",
+        "0.16.1"
     ],
-    "yanked_versions": {
-        "0.14.0": "rules_python 0.14.0 is broken due to https://github.com/bazelbuild/bazel-central-registry/issues/287, please upgrade to version >= 0.15.0"
-    }
```

Comment:
@f0rmiga Could you add this part back manually?

Reply:
Done in [9032517](https://github.com/bazelbuild/bazel-central-registry/pull/312/commits/903251778934de4695f8146c316c3ec0d20e2575).